backend/services/core/src/datasources/Events.ts

```diff
@@ -278,40 +282,37 @@ export default class EventAPI extends dbUtils.KnexDataSource {
     id: UUID,
   ): Promise<gql.Maybe<gql.Event>> {
     return this.withAccess('event:social', ctx, async () => {
-      if (!ctx.user) throw new ApolloError('Not logged in');
```

Comment:
Why where these `if (!ctx.user)` checks here to begin with?

Reply:
Either someone wrote it before adding "event:social", or event:social was incorrectly * instead of _. That's just my guess though